### ISSUE TITLE
Integrate updates from fidelity tax engine project

### ISSUE THREAD

**Issue by jwicks** [candidpartners/concourse-cicd-demo#6]
Here is what I can see off the top of my head. @gjednaszewski , @jdunnaway any others?

Structure/Make/scripts
- A separate account-level project with some shared infra deployed, eg. test data S3 bucket or KMS key
- Use of shared Makefile includes within services
- Performance jest tests and Makefile targets to standup this special perf environment

Terraform
- Use of global and regional infrastructure. There's a start to that here: https://github.com/candidpartners/concourse-cicd-demo/pull/5
- Use of TF remote state to reference infra deployed in a separate service
- Use of feature flags from config files to conditionally deploy resources

Code
- Use of jestGlobalSetup.js file and utility code to quickly clear DynamoDB or S3 before tests
- Code sample of dal.js / DAL class for DynamoDB operations
- Code sample of reusable StreamProcessorLambda class for lambdas that process DDB stream data

**Comment by gjednaszewski** [comment]
Yeah, that's a good list.  I'll try to look thorough this repo this week to refresh my memory on what else might need to be incorporated
